code/lib/cli/src/helpers.ts

```diff
@@ -206,8 +206,7 @@ const frameworkToRenderer: Record<SupportedFrameworks | SupportedRenderers, Supp
   solid: 'solid',
   svelte: 'svelte',
   sveltekit: 'svelte',
-  vue: 'vue',
-  vue3: 'vue',
+  vue3: 'vue3',
```

Comment:
I'm not sure what this map is used for, but is this change correct?

Reply:
Looks to me like this  code is only used here:
https://github.com/storybookjs/storybook/blob/48694d15a9de7e9f8a56c7d63f05063490aa8c54/code/lib/cli/src/helpers.ts#L268-L271

It's just modifying the `Configure.mdx` a little bit with some string replacement.

I think it should not mention `vue3`, but rather say `vue`

Reply:
Thanks!